todo.py

```diff
@@ -1,11 +1,31 @@
 import json
+import os
+from datetime import datetime
+
+def get_last_modified_timestamp(file_path):
+    try:
+        timestamp = os.path.getmtime(file_path)
+        return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
+    except FileNotFoundError:
+        return None
+    
+def load_tasks():
+    file_path = "tasks.json"
+    with open(file_path, "r") as file:
+        tasks = json.load(file)
```

Comment:
You should handle if file is not found or not read properly
```suggestion
    try:
        with open(file_path, "r") as file:
            tasks = json.load(file)
    except FileNotFoundError:
        print("File not found.")
    except json.JSONDecodeError:
        print("Error reading the JSON file.")
```